src/CMakeLists.txt

```diff
@@ -377,6 +377,10 @@ if (OpenMVG_USE_OPENCV)
   endif (NOT OpenCV_FOUND)
 endif (OpenMVG_USE_OPENCV)
 
+# ==============================================================================
+# Dependency libraries:
+# ==============================================================================
+add_subdirectory(dependencies/cereal)
```

Comment:
We need to do this only if we use the internal cereal. Please note that OpenMVG can use an external cereal library.
Can you check that your modification will work even you are using an external cereal library.

Reply:
Got it.